src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+> rather this _determinate_ unity, or one in which being and nothing equally
+> _are_. However, inasmuch as being and nothing are each unseparated from its
+> other, _each is not_. In this unity, therefore, _they are_, but as vanishing,
+> only as _sublated_. They sink from their initially represented
+> _self-subsistence_ into moments which are _still distinguished_ but at the
+> same time sublated (Hegel 2010, 80/21.92).
+
+import { Tabs } from "nextra/components";
+
+<Tabs items={['Main reading', 'Alternative reading']}>
+
+<Tabs.Tab data-name="main reading">
+
+Here Hegel stresses that `becoming` is _not_ to be understood as just the a
+static unity that merely combines `being` and `nothing` side-by-side without
+taking into account the nuances of their respective developments. Inasmuch as
+`becoming` is regarded as the unity of the two categories it has specifically to
+do with their inseparability.
+
+However, this inseparability needs to be further qualified. If each category is
+isolated with this inseparability, then it is negated on account that it does
+not have any self-subsistence, reality or independence vis-à-vis the other it is
+inseparable from.
+
+Therefore, _the unity of_ `becoming` contains both categories as _moments_. This
+sublated state does double work: first, it ensures the distinctiveness of each
+category; second, it _also_ illustrates their inseparability through the shared
+context.
+
+Put differently, the developments of `being` and `nothing` lead to divergent
+results: on the one hand, each is an immediate self-subsistent category totally
+distinct from the other, but, on the other hand, each is indistinguishable from
+the other and vanishes into that other, implying an undeniable inseparability.
+This looks like a problem: How can these results be coherently understood
+together? `Becoming` is the category that creatively combines these two results
+through the idea of "moment": a logic that at once joins together while it
+differentiates.
+
+</Tabs.Tab>
+
+<Tabs.Tab data-name="alternate reading">
+
+Here `becoming` is further developed into two phases. First, as the
+"unseparatedness" of `being` and `nothing`. Secondly, as the "unity" of `being`
+and `nothing`. Hegel makes this distinction within `becoming` following the
+inseparability and difference exposed in the
+[first section](#the-unity-of-being-and-nothing) since each of these signal a
+different conception with regards to how the category is understood.
+
+For example, insofar as `being` is understood to be inseparable from `nothing`,
+`being` does not have any self-subsistence, reality or independence and so its
+meaning of `pure being` is negated and with it the category itself ("each is
+not"). And, conversely, insofar as `being` is understood to be distinct from
+`nothing`, it is joined into a determinate unity with `nothing` where the two
+"equally are". In the second phase, Hegel adds additional qualifiers: each
```

Comment:
Lines 115-117. A new reader might be confused about how Being could be distinct from Nothing and (1) be in a determinate unity with it in which both (2) equally are. The idea that being in a determinate unity that offers them each independence might be sufficiently counter-intuitive to a new reader as to make the passage too confusing. Maybe an extra line might help to make it less ambiguous?

Reply:
This is a great point! 💪 I've added a paragraph highlighting the issue below. But this might be something to pick up on and further expand.